test/updateIn-spec.js

```diff
@@ -27,6 +27,12 @@ describe('u.updateIn', () => {
     expect(result).to.eql({ a: [0, 3, 0] });
   });
 
+  it('can create array if all keys are numbers', () => {
+    const result = u.updateIn('a.1', 3, null);
+
+    expect(result).to.eql({ a: [, 3] }); // eslint-disable-line no-sparse-arrays
```

Comment:
I'm actually surprised that this is valid syntax. It's not pretty though. Another option would be:
```javascript
expect(result.a[1]).to.eql(3);
```